app/views/user/_track_progress.erb

```diff
@@ -1,5 +1,5 @@
 <div class="track-info col-xs-4 col-sm-4 col-md-4 col-lg-4">
-  <%= track_icon(track_info.language, '60') %>
+  <%= track_icon(track_info.track.id, '60') %>
```

Comment:
The `track_info` had a track, but it was a weirdo track that wasn't quite a real track. I swapped that out, and the `language` was actually the `ID`. So bad :)